pkg/session/session.go

```diff
@@ -40,18 +41,59 @@ import (
 	"github.com/vmware/govmomi/vim25/methods"
 	"github.com/vmware/govmomi/vim25/soap"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
 
 	infrav1 "sigs.k8s.io/cluster-api-provider-vsphere/apis/v1beta1"
 	"sigs.k8s.io/cluster-api-provider-vsphere/pkg/constants"
 )
 
+const (
+	metricNameSpace            = "session"
```

Comment:
let's set the namsepace to `capv` or something similar.

Using `session` as namespace and `operation` as name would e.g. lead to the metric name:
`session_operation` and it would be impossible to map it back that it is coming from capv.

Reply:
I would set "capv_client_session" as sub-system (like we do in core CAPI) and not set any namespace